package.json

```diff
@@ -18,15 +18,15 @@
     "babel-preset-es2015": "^6.24.0",
     "babelify": "^7.3.0",
     "ethereumjs-blockchain": "^1.4.1",
-    "ethereumjs-testing": "https://github.com/ethereumjs/ethereumjs-testing",
+    "ethereumjs-testing": "https://github.com/ethereumjs/ethereumjs-testing#metro-byzantium",
     "ethereumjs-tx": "1.3.3",
     "level": "^1.4.0",
     "leveldown": "^1.4.6",
     "levelup": "^1.3.2",
     "memdown": "^1.1.0",
     "minimist": "^1.1.1",
     "standard": "^5.2.2",
-    "tape": "^4.2.0"
+    "tape": "4.6.3"
```

Comment:
its mentioned in the commit: to downgrade tape to hide stack trace in test runner. tape versions higher 4.6.3 print a full js stack trace on failures, but its useless for our purposes (we'd want an EVM trace, not a test runner trace). Hiding it lets the test results fit in the travisCI log.